rocky/rocky/views/ooi_detail.py

```diff
@@ -108,7 +108,7 @@ def get_task_history(self) -> Page:
 
         # FIXME: in context of ooi detail is doesn't make sense to search
         # for an object name, so we search on plugin id
-        plugin_id = self.request.GET.get("task_history_search")
+        plugin_id = self.request.GET.get("task_history_search") or None
```

Comment:
Seems a bit redundant, since `QueryDict.get` returns `None` by default. See [this](https://docs.djangoproject.com/en/4.2/ref/request-response/#django.http.QueryDict.get)

```suggestion
        plugin_id = self.request.GET.get("task_history_search")
```